tests/web3js/eth_non_interactive_test.js

```diff
@@ -94,8 +94,11 @@ it('get transaction', async() => {
 })
 
 it('get status', async() => {
-    //let mining = await web3.eth.isMining() todo
-    let sync = await web3.eth.isSyncing()
-    //assert.isFalse(mining)
-    assert.isFalse(sync)
+    let mining = await web3.eth.isMining()
+    assert.isFalse(mining)
+
+    let syncInfo = await web3.eth.isSyncing()
+    assert.equal(syncInfo.startingBlock, '0x3')
```

Comment:
starting block should be 1

Reply:
> starting block should be 1

This is actually from the JS config file:
```js
module.exports = {
    web3: web3,
    eoa: web3.eth.accounts.privateKeyToAccount("0xf6d5333177711e562cabf1f311916196ee6ffc2a07966d9d4628094073bd5442"),
    fundedAmount: 5.0,
    startBlockHeight: 3n, // start block height after setup accounts
    serviceEOA: "0xfacf71692421039876a5bb4f10ef7a439d8ef61e", // configured account as gw service
    successStatus: 1n
}
```
starting block height is set to `3n`.

> also don't hardcode values, better to get the latest block by calling web3 latest block function

Good point :+1: Updated in https://github.com/onflow/flow-evm-gateway/pull/180/commits/1405e2c8898b45563a6abaa55c3ee55757fc55d0